renderer/components/Player/Player.svelte

```diff
@@ -48,6 +49,7 @@
           player.load()
         }
       } else {
+        invoke('playlists.prependSingleMerge', 151120, current.id)
```

Comment:
Thanks to `tracks.addRecent()` you won't need anymore to hardcode the Recent model id on the UI